dart/2.Dart_basics/Excercises/4_solution.md

```diff
@@ -0,0 +1,30 @@
+# Solution 4
```

Comment:
It's cool but this exercise give me the idea of making an encryption machine as exercise: 

1. you replace one vowel for a symbol.
2. you could enter a text and will be 'encrypted' or viceversa.

What do you think?

Reply:
great! I'll write it :)

Reply:
I love that enthusiasm!